eq-author/src/App/settings/ThemesPage.js

```diff
@@ -87,7 +91,7 @@ const EqIdInput = ({ eqId = "", questionnaireId, shortName }) => {
     value = value.trim();
     updateQuestionnaireTheme({
       variables: {
-        input: { questionnaireId, shortName, eqId: value },
+        input: { questionnaireId, shortName, eqId: value.trim() },
```

Comment:
There's already a `value.trim()` a couple of lines above, so you don't need to do it again here.

```suggestion
        input: { questionnaireId, shortName, eqId: value },
```

Reply:
Haha, oops. I thought I'd deleted the `value = value.trim()` line. Will push in a sec, cheers.